src/hearings/containers/cancel-hearing/cancel-hearing.component.html

```diff
@@ -2,9 +2,43 @@
 <div class="govuk-width-container">
   <div class="page">
     <main id="content" role="main">
-      <p>
-        cancel-hearing works!
-      </p>
+      <a href="javascript:void(0)" class="govuk-back-link">Back</a>
+
+      <div class="govuk-grid-row">
+        <xuilib-hmcts-error-summary [errorMessages]="validationErrors" [header]="'There is a problem'"></xuilib-hmcts-error-summary>
+
+        <div class="govuk-grid-column-two-thirds">
+          <form class="form" [formGroup]="hearingCancelForm">
+            <h1 class="govuk-heading-l">
+                Are you sure you want to cancel this hearing?
+            </h1>
+            <exui-hearing-parties-title></exui-hearing-parties-title>
+            <div class="govuk-form-group">
+              <fieldset class="govuk-fieldset">
+                  <legend class="govuk-fieldset__legend govuk-fieldset__legend--s">
+                      <h1 class="govuk-fieldset__heading">
+                          Why is this hearing being cancelled?
+                      </h1>
+                  </legend>
+                  <div id="other-panel-roles-hint" class="govuk-hint">
```

Comment:
is this panel? the name of id exactly matched what this page?

Reply:
removed id. Sorry, its copied from mockup